scripts/cc-cli/src/commands/setKeys.ts

```diff
@@ -39,7 +39,7 @@ async function setKeysAction(options: OptionValues) {
     keys = options.keys;
   }
 
-  const tx = api.tx.session.setKeys(keys, []);
+  const tx = api.tx.session.setKeys(keys, "");
```

Comment:
@jonZlotnik you had some concerns about this on Slack. Can you check it out in the context of this PR?

Reply:
This was checked, there is no check for ownership proof for setKeys